helpers/helperFunctions.js

```diff
@@ -22,4 +29,70 @@ exports.getFieldsFormApi = function (bodyStr) {
 		Latitude: item.venue.location.lat,
 		Longitude: item.venue.location.lng
 	}))
-}
\ No newline at end of file
+}
+
+// get inputs data
+
+exports.processInputsData =  function (data) {
+    try {
+      return {
+              venueType:  validations.validateName(data.query),
+              radius:     validations.validateRadius(data.radius),
+              latitude:   validations.validatePoint(data.latitude),
+              longitude:  validations.validatePoint(data.longitude)
+      }
+      
+   
+    }
+    catch (err) {
+          const expectedInput = 'please input correct data';
+          throw new Error('Incorrect input parameter: '+ err.message +'\nExpected:\n' + expectedInput);
+      }
+  }
+  
+
+  //get data from Foursquare API
+
+
+ exports.getFoursquareApiData =  function(data) {
+
+    return new Promise (function(res,rej){
+        request({
+        url: 'https://api.foursquare.com/v2/venues/explore',
+        method: 'GET',
+        qs: {
+            client_id: config.CLIENT_ID,
+            client_secret: config.CLIENT_SECRET,
+            ll: data.latitude + ',' + data.longitude,
+           radius: data.radius,
+            query: data.venueType,
+            v: getCurrentDate()
+        }
+        }, (err, r, body) => {
+          err ? rej(err) : res(getFieldsFormApi(body));
+        });
+    
+    })
+    
+      
+    }
+    
+    exports.responseCsvVenue = function ( data ) {
+
+        const columnNames = ["Name", "City", "Street", "Latitude", "Longitude"];
+
+
+        const json2csvParser = new Json2csvParser({ columnNames,quote: ' ' });
+        const csv = json2csvParser.parse(data);
+        
+          const responseCsv = {
+            statusCode: 200,
+            headers: {'Content-Type' : 'text/csv; charset=utf-8'},
+            body: JSON.stringify({
+            message: csv
```

Comment:
next PR - to change this

https://github.com/shcherbyakdev/find-places/issues/1